scripts/testnet/sovereignBridge/config/deploy.snippets.sh

```diff
@@ -0,0 +1,125 @@
+# This function will:
+# - deploy all main chain contracts and update sovereign configs
+# - deploy sovereign nodes with all services
+deploySovereignWithCrossChainContracts() {
+    deployMainChainContractsAndSetupObserver || return
+
+    sovereignDeploy
+}
+
+# This function will:
+# - deploy all main chain contracts
+# - update sovereign configs
+# - prepare a main chain observer for sovereign nodes
+deployMainChainContractsAndSetupObserver() {
+    checkWalletBalanceOnMainChain || return
```

Comment:
Shouldn't we do a` || return` for each of the functions below?

Reply:
I added more `|| return` for some more functions that can actually fail because of missing stuff.